spinoso-time/src/time/mod.rs

```diff
@@ -1,19 +1,29 @@
 //! Implementations of Ruby [`Time`], a timezone-aware datetime.
 //!
-//! This module contains implementations of a timestamp storage struct and
```

Comment:
"this module" refers to `spinoso_time::time`, which contains several `time` implementations.

Is there a clearer way for us to say that? Does it make sense to revert this change?

Reply:
Ahhh I see. This was just my bad understanding of "this" - but reading it again actually makes sense with the prior text.